server/cmd/mmctl/commands/compliance_export.go

```diff
@@ -21,17 +24,59 @@ var ComplianceExportListCmd = &cobra.Command{
 	RunE:    withClient(complianceExportListCmdF),
 }
 
+var ComplianceExportShowCmd = &cobra.Command{
+	Use:     "show [complianceExportJobID]",
+	Example: "  compliance_export show o98rj3ur83dp5dppfyk5yk6osy",
```

Comment:
Why do we need the leading spaces?

Reply:
I was following what was done for the export cmd, but I guess we don't really need that (it only affects the docs). 🤷   Fixed.